### PR TITLE
[compliance] Package security-agent-policies compliance configs

### PR DESCRIPTION
**Pull request by xornivore**
### What does this PR do?

Packaging default compliance benchmarks from a security-agent-policies repo in agent packages,

This mimics pulling a version of integrations-core, there is however no install to do - just a file copy. 

### Motivation

Meaningful configuration out of the box for compliance.

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Build agent images, confirm benchmarks are correctly packaged under `/etc/datadog-agent/compliance.d`

### CODE REVIEW

**Review by KSerrania — COMMENTED** [review]
I think you'll want to declare joint ownership between Agent Platform and your team in `CODEOWNERS` for the new `omnibus/config/software/datadog-security-agent-policies.rb` file, same as what we do here: https://github.com/DataDog/datadog-agent/blob/c879895b994a6d1ddc3826397b7aec25a0790add/.github/CODEOWNERS#L76-L77

(otherwise you'll need a review from us every time you want to change something in this software definition).

File: tasks/cluster_agent.py
Comment (ahmed-mez) [inline tasks/cluster_agent.py:47]: should we make this configurable? i.e could it be useful to package a custom policies version to build a custom agent version for example?

File: tasks/cluster_agent.py
Comment (xornivore) [reply tasks/cluster_agent.py:50]: Oh, great, yes, that's much better, thank you! I believe it should also cover @ahmed-mez's comment about making it configurable - so the release-version is a param you can use that as well.

File: tasks/cluster_agent.py
Comment (xornivore) [reply tasks/cluster_agent.py]: Yes, definitely. Made it configurable in the task.

**Review by clamoriniere — APPROVED** [review]

**Review by prognant — APPROVED** [review]
LGTM

**Review by KSerrania — APPROVED** [review]

**Pull request merged**
